test-integration/config.test.js

```diff
@@ -60,7 +60,7 @@ fs.readdirSync(path.resolve(__dirname, './config')).forEach(filename => {
         const kongState = ignoreConfigOrder(await readKongApi(testAdminApi));
 
         expect(getLog()).toMatchSnapshot();
-        expect(exportToYaml(kongState)).toMatchSnapshot();
+        expect(exportToYaml(ignoreKeys(kongState))).toMatchSnapshot();
```

Comment:
The `ignoreKeys` function makes sure we don't snapshot UUIDs (which are determined by kong at runtime) or properties we expect to change, like timestamps. It was previously only applied to entries in the logs, but we also need to handle changes to UUIDs in the config for things like routes or the anonymous consumer id.